### PR TITLE
Samples DNN: tf_text_graph_sd.py loads box coder variance and box NMS…

### LINKED ISSUE #15617: DNN: tf_text_graph_ssd.py generated pbtxt leads to wrong results
##### System information (version)

- OpenCV => 4.1.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2017
- Inference engine 2019.2.275

##### Detailed description
I used the version of script currently available in the master branch. My model was generated with TF 1.14 and is a ssd mobilenet v2 network. The first image is the result of the inference using the intel inference engine with the optimized version of the net. It is correct (icons should be detected) and similar to what is detected using TF. On the right, the detection using the pbtxt file with OpenCV backend (same result with IE), which is clearly wrong. 

image

image

Files: 
[inference.zip](https://github.com/opencv/opencv/files/3675467/inference.zip)

##### Steps to reproduce
```
python ~/opencv/samples/dnn/tf_text_graph_ssd.py --input frozen_inference_graph.pb --config pipeline.config --output graph.pbtxt
```
Output: 
Scale: [0.050000-0.500000]
Aspect ratios: [1.0, 3.0, 10.0]
Reduce boxes in the lowest layer: True
Number of classes: 6
Number of layers: 6
box predictor: convolutional
Input image size: 800x800

Steps for getting optimized model:
```
python ~/openvino_2019.2.275/deployment_tools/model_optimizer/mo_tf.py --input_model frozen_inference_graph.pb --tensorflow_use_custom_operations_config ~/openvino_2019.2.275/deployment_tools/model_optimizer/extensions/front/tf/ssd_support_api_v1.14.json --tensorflow_object_detection_api_pipeline_config pipeline.config --input_shape [1,800,800,3]
```

### PR DESCRIPTION
**Pull request by lorenzolightsgdwarf**
… parameters from config file

### This pullrequest changes
Before these values were set to the default values described in the object detection API protos.
The change reflects what is done in the Intel Openvino at deployment_tools\model_optimizer\extensions\front\tf\ObjectDetectionAPI.py

resolves #15617

### CODE REVIEW

File: samples/dnn/tf_text_graph_ssd.py [hunk 1]
```diff
@@ -303,7 +306,10 @@ def addConcatNode(name, inputs, axisNodeName):
 
         priorBox.addAttr('width', widths)
         priorBox.addAttr('height', heights)
-        priorBox.addAttr('variance', [0.1, 0.1, 0.2, 0.2])
+
+        anchor_generator = config['anchor_generator'][0]
```
Comment (dkurt) [inline samples/dnn/tf_text_graph_ssd.py]: Unused?

Comment (lorenzolightsgdwarf) [reply samples/dnn/tf_text_graph_ssd.py]: You are right

**Comment by dkurt** [comment]
Please do not create master-to-master PRs. Otherwise it breaks your fork in case of updates. It's more convenient to create a new branch like
```
git checkout -b dnn_fix_tf_ssd
```

I'd like to ask you to change the target branch to 3.4. We will port the changes to the master branch as well. Thanks in advance!

**Comment by lorenzolightsgdwarf** [comment]
Sorry, I´m not very familiar with the process and I should have read the guidelines more carefully.
Regarding the branching, can I leave it to master for the moment? I don´t mind about my fork because I did it only for the purpose of fixing this issue then I will delete it.

**Comment by lorenzolightsgdwarf** [comment]
Ok I completely messed up. I will close this and open a new one branching from 3.4

**Comment by dkurt** [comment]
Don't worry. It's just to keep both branches as compatible as possible. Thanks for contribution!

**Pull request closed without merging**